api/Makefile

```diff
@@ -11,7 +11,7 @@ POETRY_VERSION ?= 1.8.3
 
 GUNICORN_LOGGER_CLASS ?= util.logging.GunicornJsonCapableLogger
 
-SAML_REVISION ?= v1.6.3
+SAML_REVISION ?= feat/jwt-cookie-auth
```

Comment:
Remove this once [this PR](https://github.com/Flagsmith/flagsmith-saml/pull/47) is merged.